admin/commands/management/add_external.go

```diff
@@ -57,7 +57,7 @@ type AddExternalCommand struct {
 	RunsOnNodeID        string            `name:"agent-node-id" help:"Node ID where agent runs (default is autodetected)"`
 	Username            string            `help:"External username"`
 	Password            string            `help:"External password"`
-	CredentialsSource   string            `type:"existingfile" help:"Credentials provider"`
+	ServiceParamsSource string            `help:"Path to file with service parameters"`
```

Comment:
Let's please update the help here so it's clear what is expected.
With the current help it's not clear what the file shall contain.